### DISCUSSION TITLE
Erro no cors em modo debug. FullStack

### DISCUSSION THREAD

**Discussion by tiaotj** [balta-io/forum#717] [Tira Dúvidas] [answered]
Estou com um problema ao executar o projeto .web via debug, 
![image](https://github.com/user-attachments/assets/25cfa7a2-a84a-4bc0-9cb9-7c9d63c46b92)
Fala que não o endereço está sem permissão
Mais já coloquei todas as portas possível no BuilderExtensio
![image](https://github.com/user-attachments/assets/d025eb63-12d1-4ac6-a094-39eefb9110c8)
O estranho que isso só acontece se eu executo como debug. Se eu rodar o projeto no powershell, com dotnet run ele funciona normalmente. 
O problema que preciso executar como debug. pra acompanhar o andamento do código

**Comment by andrebaltieri** [comment, marked as answer]
Duplicado:
https://github.com/balta-io/forum/discussions/636#discussioncomment-10136288
